lib/views/widgets/home_page.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({ Key? key }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Stack(
+          children: [
+            Container(
+              height: 250,
+              decoration: const BoxDecoration(
+                  gradient: LinearGradient(
+                    begin: Alignment.topRight,
+                    end: Alignment.bottomLeft,
+                    colors: [
+                      AppColor.light,
+                      AppColor.primary,
+                    ],
+                  ),
+                // color: Color(0xFFDCE3F9),
+                borderRadius: BorderRadius.vertical(
+                  bottom: Radius.elliptical(160, 20),
+                ),
+              ),
+            ),
```

Comment:
@ErickDoroy make this a reusable widget since it's gonna be used on most pages sa app

Reply:
same thoughts with Ms. Aubrey.. was thinking about just calling the layout as Template and then provide a child to it.. something like
`Template (
  child: [Content here]
)`